docs-source/content/quickstart/get-images.md

```diff
@@ -25,7 +25,7 @@ and accept the license agreement for the [WebLogic Server image](https://hub.doc
 1. Pull the Traefik load balancer image:
 
     ```bash
-    $ docker pull traefik:1.7.12
+    $ docker pull traefik:2.2.1
```

Comment:
@ankedia, what version did you test with? We say below "2.2.1 and later", but it seems like we ought to recommend the most recent version that we've validated.

Reply:
I tested with version Traefik 2.2.1